.eslintrc

```diff
@@ -3,12 +3,13 @@
   "extends": [
     "plugin:react/recommended",
     "plugin:@typescript-eslint/recommended",
-    "plugin:prettier/recommended"
+    "plugin:prettier/recommended",
+    "plugin:jsx-a11y/recommended"
```

Comment:
@jhpedemonte seemed to have utilized the recommended settings before. When checking to see which settings CRA uses by default (the root of why this issue was opened) I'm unsure whether these would conflict, see what I presume to be the CRA defaults [here](https://github.com/facebook/create-react-app/blob/master/packages/eslint-config-react-app/index.js#L285)

This made me question whether we should just attempt to use this [package](https://github.com/facebook/create-react-app/tree/master/packages/eslint-config-react-app) at the root of our repository to make sure that we are using CRA defaults across the board and don't see similar CI/CD failures